published/external/xdp/program.h

```diff
@@ -38,6 +38,8 @@ typedef enum _XDP_MATCH_TYPE {
     XDP_MATCH_IP_NEXT_HEADER,
     XDP_MATCH_INNER_IPV4_DST_MASK_UDP,
     XDP_MATCH_INNER_IPV6_DST_MASK_UDP,
+    XDP_MATCH_ICMPV4_ECHO_REPLY_IP_DST,
+    XDP_MATCH_ICMPV6_ECHO_REPLY_IP_DST
```

Comment:
Nit: still need a trailing `,` on the final enum entry. This keeps the diff cleaner when the next entry is added.

Reply:
fix pending